3-UsingAPIs/Week1/homework/ex5-rollDiceChain.js

```diff
@@ -0,0 +1,51 @@
+// Challenge: throw five dices in sequence
+const rollDice = require('../../helpers/pokerDiceRoller');
+
+function rollTheDices() {
+  const results = [];
+  // TODO: expand the chain to include five dices
+  rollDice(1)
+    .then((value) => {
+      results.push(value);
+      return rollDice(2);
+    })
+    .then((value) => {
+      results.push(value);
+      return rollDice(3);
+    })
+    .then((value) => {
+      results.push(value);
+      return rollDice(4);
+    })
+    .then((value) => {
+      results.push(value);
+      return rollDice(5);
+    })
+    .then(() => console.log('Resolved!', results))
+    .catch((error) => console.log('Rejected!', error.message));
+}
+rollTheDices();
+//===================================================================
+const rollDiceDry = require('../../helpers/pokerDiceRoller');
```

Comment:
So this was the DRY version that we worked out together. Next time, as we discussed, make sure that you add a comment to that effect. If another mentor would review your PR, she might mistakenly conclude that this code was completely yours alone.